### ISSUE TITLE
Add offset argument to limit

### ISSUE THREAD

**Issue by pauldix** [influxdata/flux#696]
We want to be able to paginate through result sets. For example if we're pulling back distinct tag values, we'd like to be able to do this:

```
from(db:"db")
    |> range(start:-5000h)
    |> filter(fn:(r) => r._measurement == "cpu")
    |> group(by:["host"])
    |> distinct(column:"host")
    |> group(none:true)
    |> limit(n:100, offset:300)
```

@121watts will need this to really complete the schema explorer

**Comment by prashanthjbabu** [comment]
Any updates on when offset support would come to flux?

**Comment by russorat** [comment]
@prashanthjbabu thanks for your nudge. offset has been added to `limit`. https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/transformations/limit/

**Comment by prashanthjbabu** [comment]
@russorat Thanks for the update ! Is this available in flux 0.5 for Influx 1.7.x ? Reference : https://docs.influxdata.com/flux/v0.50/introduction/flux-vs-influxql/
